swagger/components/CustomLayout.css

```diff
@@ -27,4 +27,8 @@
   .swagger--head-card {
     top: 0;
   }
+}
+
+td.col.response-col_status {
```

Comment:
Some areas in request were not matched by this rule. We should change it to:

```css
tr > *.col {
    width: auto;
}
``` 
@libremente

Reply:
Ok, let me push that and squash @sebbalex 👍